spec/src/main/asciidoc/chapters/language/expressions.adoc

```diff
@@ -502,13 +493,11 @@ In the extended language, the right operand of an equality or inequality operato
 
 [WARNING]
 ====
-When using NoSQL databases, there are limitations to the support of equality and inequality operators:
-
-1. **Key-Value Databases**: Support for the equality restriction on the key attribute is required. The key attribute is defined by the annotation `jakarta.nosql.Id`. Key-value databases are not required to support any other restrictions.
+When a Jakarta Query implementation targets a non-relational database, support for equality and inequality operators might vary from what is described above.
+This specification does not require support for use of an equality or inequality operator if the underlying datastore does not provide this operator in its native querying capabilities.
+For example, some non-relational databases might only allow lookup by key, and in such databases only the entity <<structures-and-records,identifier>> may be used in an equality comparison expression.footnote:[In Jakarta NoSQL, the key attribute is identified by the annotation `jakarta.nosql.Id`.]
 
-2. **Wide-Column Databases**: Support for equality restriction and the inequality restriction on the `Id` attribute is required. Support for restrictions on other entity attributes is not required. These operations typically work only with the `Id` by default but might be compatible for other entity attributes if secondary indexes are configured in the database schema.
-
-3. **Graph and Document Databases**: Support for all equality and inequality operators is required.
+**TODO** Why does Jakarta Data not have a similar caveat applying to `between`, `like`, `in`, and  `is null`? Do we need to generalize the language above to other sorts of conditional expressions?
```

Comment:
@otaviojava 👀

Note this TODO. Don't we need to say something about support for these other operators?